src/app/pages/vm/vm-cards/vm-cards.component.ts

```diff
@@ -522,12 +523,22 @@ export class VmCardsComponent implements OnInit,OnDestroy {
     }
      else {
       if(poweroff){
-        eventName = "VmPowerOff";
-      } else {
-        eventName = "VmStop";
+         this.off_text =  `Do you wish to Power Off ${vm.name} ?`;
+      }
+      else {
+        this.off_text =  `Do you wish to Turn Off ${vm.name} ?`;
```

Comment:
As above, although I don't see what the difference is between "power off" and "turn off".  I suspect this second one should just be "stop".
`` this.off_text = Stop virtual machine "${vm.name}"?;` ``